src/quo2/components/avatars/account_avatar.cljs

```diff
@@ -27,12 +22,11 @@
     20 11))
 
 (defn account-avatar
-  [{:keys [size icon]
-    :or   {size 80
-           icon :main-icons/placeholder}}]
-  (let [icon-color           (if (theme/dark?)
-                               (:dark icon-color-value)
-                               (:light icon-color-value))
+  [{:keys [size icon color]
+    :or   {size  80
+           icon  :main-icons/placeholder
```

Comment:
The keyword qualification should be just `:i`. https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#icons

Please, let us know if you think anything could be improved in the guidelines.

Reply:
I was not aware of those guidelines, thanks for sharing!

I replaced it in 415173139a52d171e755945da1f3018ca7b111cc

This use of  `:main-icons` was already in develop (I didn't added it), Must we always replace those appearances?

Reply:
You're right about that, I actually thought we recently eliminated all references to the old icon qualification.

I understand it's not totally obvious from the outside, but we've been trying to establish some stronger conventions in the code, such as the icon keyword qualification. Right now coding in the Status app is a bit of a minefield in terms of knowing what's old or not (esp. for non-core collaborators), but we're trying to document the new guidelines as much as possible. Feedback is highly appreciated :)